src/pages/_app.js

```diff
@@ -118,6 +119,7 @@ function MyApp({
                                 </Head>
                                 <ReactQueryDevtools initialIsOpen={false} />
                                 <Navigation activeView={pathname} />
+                                <PageSpacer />
```

Comment:
Since `PageSpacer` is not needed for smaller screens, we can use <Hidden smDown> <PageSpacer/><Hidden> here instead of setting height and width to 0 in `pageSpacer/index.js`.

Reply:
I'm not sure what change you'd like me to make here.

Reply:
It was just a suggestion. The comment didn't get formatted properly. Comment updated now.

Reply:
Thanks for the correction.